.eslintrc.js

```diff
@@ -43,8 +53,6 @@ module.exports = {
         '@typescript-eslint/no-shadow': 'error',
         'no-unused-vars': 'off',
         '@typescript-eslint/no-unused-vars': ['error'],
-        'no-else-return': 'off',
```

Comment:
To me it is more clear if alternative but equally important return paths are indented same way:
```
if (x) {
    return 'foo'
} else {
    return 'bar'
}

```
If this not allowed, one return path looks like "primary" path and another is a fallback:
```
if (x) {
    return 'foo
}
return 'bar'
```

Reply:
Meh, you could still say the bit in the `if` block is the "primary", and the `else` is the fallback. IMO when faced with a low-value stylistic choice a good deciding metric is to just pick the style which minimise the amount of indentation.

I wrote a bit about avoiding `else` blocks here: https://blog.timoxley.com/post/47041269194/avoid-else-return-early